CHANGELOG.md

```
@@ -158,3 +159,4 @@ Other guiding principles:
 [#1013]: https://github.com/pragma-org/amaru/pull/1013
 [#1024]: https://github.com/pragma-org/amaru/pull/1024
 [#1027]: https://github.com/pragma-org/amaru/pull/1027
+[#1027]: https://github.com/pragma-org/amaru/pull/1029
```

markdownlint-cli2 (0.22.1)

[warning] Link and image reference definitions should be needed
Duplicate link or image reference definition: "`#1027`"

(MD053, link-image-reference-definitions)